wasm/__init__.py

```diff
@@ -620,13 +619,9 @@ def spec_validate_exportdesc(C, exportdesc):
             raise Exception("invalid")
         return ["mem", C["mems"][x]]
     elif exportdesc[0] == "global":
-        # print("global")
-        # print(len(C["globals"]),x)
         if len(C["globals"]) <= x:
             raise Exception("invalid")
         mut, t = C["globals"][x]
-        # print(mut)
-        # if mut != "const": raise Exception("invalid") #TODO: this was in the spec, but tests fail linking.wast: $Mg exports a mutable global, seems not to parse in wabt
```

Comment:
Is there an issue tracking this TODO warning? Seems like something that deserves further research.